src/app/collection/shared/included/collection-learning-object-card/collection-learning-object-card.component.html

```diff
@@ -0,0 +1,29 @@
+<div class = "container">
+
+
+    <div class = "header" [ngClass] = "{'unit':learnObj.length==='unit', 'nanomodule':learnObj.length==='nanomodule'
```

Comment:
To make this shorter you can do:
```suggestion
    <div class = "header" [ngClass] = "learnObj.length">
```

Then you can conditionally set the styles in the scss file.

Reply:
I would still need the overlay class still for the grey overlay that goes on top of the header. The header just contains the image

Reply:
The styles can include the image and overlay.